### ISSUE TITLE
New install failed when executed under Fish shell

### ISSUE THREAD

**Issue by romaninsh** [pforret/macos_m1_cli#3]
With existing brew (rosetta 2), got this error (i'm using `fish`):

```
$ ./m1_homebrew  -a arm64 install
Program: m1_homebrew 1.3.1
✔  MacOS version = 11.1
✔  Script running in i386 mode
Are you sure you want to install Homebrew for arm64 to /opt/homebrew? [y/N] y
…  Create folder for [/opt/homebrew] Homebrew ...

Password:
Sorry, try again.
Password:
…  Download Homebrew into [/Users/rw/.homebrew] ...
✔  After download: Homebrew folder is 15 MB
✔  After update  : Homebrew folder is 64 MB
✔  After install : Homebrew folder is 464 MB
…  Move Homebrew files to /opt/homebrew
Create folder share/zshomebrew/SECURITY.md ...
Create folder share/zsh/site-functions
Create folder Caskroom
✔  All files moved to /opt/homebrew
…  Add the following to your shell startup script, could not be done automatically
#####
./m1_homebrew: line 192: ARCH: unbound variable
✖ m1_homebrew: ERROR 1 after 136 seconds
> last command : 'startup_config "$ARCH" "$HOMEBREW_PREFIX"'
${error_prefix} from line 1 : #!/usr/bin/env bash
```

**Comment by romaninsh** [comment]
Looks like

```
startup_config "$ARCH" "$HOMEBREW_PREFIX"
```

should have `$architecture`

startup script should go into `~/.config/fish/conf.d/m1_homebrew.fish`

finally adding to $PATH in fish (non-permanent):

```
[ (arch) = "airm64" ] &&  set -gx PATH /opt/homebrew/bin /opt/homebrew/sbin $PATH 
```

**Comment by romaninsh** [comment]
Nice. And thanks for this amazing script!

**Comment by pforret** [comment]
Hi Roman,

to what config file should I add this line "[[ (arch) == "arm64" ]] &&  set -gx PATH /opt/homebrew/bin /opt/homebrew/sbin $PATH " ? `~/.config/fish/config.fish` ?

(cf [unix.stackexchange.com/questions/211519/how-to-edit-the-fish-shell-startup-script](https://unix.stackexchange.com/questions/211519/how-to-edit-the-fish-shell-startup-script) )

Peter

**Comment by pforret** [comment]
I fixed the `$architecture` error in v1.3.2!
